pkg/budget/budget.go

```diff
@@ -29,6 +29,8 @@ func (budgetSvc *AWSBudgetService) SetCostExplorer(costExplorer awsiface.CostExp
 // Implement the CalculateTotalSpend method of the Service interface
 func (budgetSvc *AWSBudgetService) CalculateTotalSpend(startDate time.Time, endDate time.Time) (float64, error) {
 
+	const foo = "bar"
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
U1000: const `foo` is unused (unused)